mmv1/products/dataform/Repository.yaml

```diff
@@ -78,6 +80,8 @@ examples:
     # See : https://github.com/hashicorp/terraform-provider-google/issues/17335
     # See : https://issuetracker.google.com/issues/287850319
     exclude_docs: true
+    # These tests create a new KMS key ring and key with each run, which cannot be deleted.
+    exclude_test: true
```

Comment:
```suggestion
```
Can't exclude every test or the build fails, so let's just exclude the one that's creating new keys for now